### PR TITLE
update about page

### PR DESCRIPTION
**Pull request by armanbance**

### CODE REVIEW

**Comment by armanbance** [comment]
Screenshot 2024-08-16 at 6 17 00 PM

Screenshot 2024-08-16 at 6 15 48 PM

Screenshot 2024-08-16 at 6 17 00 PM

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @tailwind base;
 @tailwind components;
```
Comment (evanugarte) [inline src/index.css]: undo the changes to this file

**Comment by armanbance** [comment]
Screenshot 2024-08-16 at 6 16 28 PM

File: src/input.css [hunk 2]
```diff
@@ -1,3 +1,3 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+@tailwind utilities;
```
Comment (evanugarte) [inline src/input.css]: this one too, add back the newline at the bottom

**Comment by akshtshrma24** [comment]
oh what a nice page this is

**Review by akshtshrma24 — APPROVED** [review]

**Review by evanugarte — APPROVED** [review]
